rotkehlchen/tests/unit/test_protocol_balances.py

```diff
@@ -342,3 +354,85 @@ def test_gmx_balances(
             usd_value=FVal('42.939657170821103766867050208907'),
         ),
     }
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [[
+    '0xecdED8b1c603cF21299835f1DFBE37f10F2a29Af', '0x8Cbac427B6967d5d84Ec4230081e2763AB3A8C92',
+]])
+def test_aave_v3_balances(blockchain: 'ChainsAggregator') -> None:
+    """Check that Aave v3 positions/liabilities are properly returned in the balances query"""
```

Comment:
```suggestion
    """Check that ethereum mainnet Aave v3 positions/liabilities are properly returned in the balances query"""
```

Also would need one for multiple chains. Either same or other test